### ISSUE TITLE
Database tick: 0 does not match supplied tick: 2000 (after release 0.14)

### ISSUE THREAD

**Issue by gennaro-tedesco** [bvaisvil/zenith#126]
After the latest release 
```
zenith -V
zenith 0.14.0
```
invoking `zenith` results in an exception
```
Database tick: 0 does not match supplied tick: 2000.
Proceed with (D)atabase tick: 0, (S)upplied tick with history disabled: 2000, (E)xit?:
```
independent of the arguments given.

My architecture is the following, if it helps
```
Machine:    MacBookPro18,1 
Kernel:     Darwin 22.3.0  
OS:         macOS 13.2.1 Ventura
Terminal:   iTerm2 (Version 3.4.19)
```

**Comment by gennaro-tedesco** [comment]
I confirm that deleting `Library/Caches/zenith/store` works for the moment :)

**Comment by bvaisvil** [comment]
Great, you wouldn't happen to have a copy of that file still? Would be helpful in tracking down the issue.

**Comment by gennaro-tedesco** [comment]
Unfortunately not :(

**Comment by noyez** [comment]
@bvaisvil  I have the same issue, and still have the DB file, its 228Mb, if you'd like it, let me know.

**Comment by bvaisvil** [comment]
> @bvaisvil  I have the same issue, and still have the DB file, its 228Mb, if you'd like it, let me know. 

Yes, please! It should compress rather well. Hopefully small enough to add to the issue.

**Comment by noyez** [comment]
> > @bvaisvil  I have the same issue, and still have the DB file, its 228Mb, if you'd like it, let me know.
> 
> Yes, please! It should compress rather well. Hopefully small enough to add to the issue.

Sorry for the delay, here is the gzip'd zenith store, which casuses the error in the original issue above. 

[zenith_store_currupt_store.gz](https://github.com/bvaisvil/zenith/files/11162795/zenith_store_currupt_store.gz)

**Comment by bvaisvil** [comment]
Thank you!

**Comment by TheSirC** [comment]
I have one too, are you interested ?

**Comment by bvaisvil** [comment]
Yes, that would be very helpful.

**Comment by TheSirC** [comment]
Here it is : sorry for the long response
[store.gz](https://github.com/bvaisvil/zenith/files/11547318/store.gz)

**Comment by bvaisvil** [comment]
No worries. I've been super busy, myself. Thank you very much for the file.
